e2e/scripts/oebs_aks.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+node_count="2" #--- number of nodes for the Azure kubernetes cluster
+node_vm_size="Standard_A2_v2" #--- VM size for the Azure Kubernetes  cluster
```

Comment:
Would recommend to take user inputs with defaults for the node count and vm type.

Reply:
@nsathyaseelan Refer https://unix.stackexchange.com/a/122848